### ISSUE TITLE
Status View of Localnet, add to CI for testing

### ISSUE THREAD

**Issue by ClaytonNorthey92** [hemilabs/heminetwork#97]
### Confirmation

- [X] I have checked for similar issues.

### Problem

As of now, when you run localnet, _a lot of setup is occurring_.  It's hard to find out what is going right or wrong.

Create a "view" that allows us to see the status of localnet.  This should have two formats: human-readable and json. With the json format, we can run automated tests against the state of localnet.

The human readable can look something like this (not complete yet):
```
+----------+-------------+
| bitcoind | not running |
| bfgd     | not running |
| bssd     | not running |
+----------+-------------+
```

#### First ETH block containing Batcher publication
First L1 ETH block `from` batcher address `to` batcher inbox address

#### Most recent ETH block containing Batcher publication
same as above, but most recent

#### number of Batcher publications
a count of the above

#### Total $HEMI paid out for PoP
a balance query of the PoP Miner address to the GovernanceToken contract

#### Total $ETH deposited/withdrawn (once we get that working in e2e testing)
not yet

again, using the json output, we can use this in automated testing and CI

### Suggested solution

use go pretty https://github.com/jedib0t/go-pretty

**Comment by ClaytonNorthey92** [comment]
I am putting this in review, though there is no PR yet, just confirming with @max-sanchez on final test cases, but this is running in CI on a branch.  You can see the results here https://github.com/hemilabs/heminetwork/actions/runs/8883094859/job/24389209865#step:6:14

**Comment by joshuasing** [comment]
I believe this was resolved by #106, please reopen the issue if this is wrong.
